### ISSUE TITLE
Password reset link doesn't work

### ISSUE THREAD

**Issue by mbklein** [samvera-labs/nurax#1]
### Descriptive summary

After entering my email address and clicking _Send me password reset instructions_, I get the following error:

![image](https://user-images.githubusercontent.com/196872/225427071-35938a34-cf23-495b-a274-ab5afce4121d.png)

This happens on all three instances – dev, stable, and pg

### Rationale

Provide the rationale or user story that describes "why" this issue should be addressed. Especially if this is a new feature or significant change to the existing implementation.

### Expected behavior

Password reset request sends a password reset email

### Actual behavior

See above

### Steps to reproduce the behavior

1. Click *Login*
2. Click *Forgot your password?*
3. Enter an email address and submit the form

**Comment by jlhardes** [comment]
I don't think we have ever had the nurax instances set up to send emails, so this might be fine or at least not different than before?
